docs/configuration/backends/docker.md

```diff
@@ -71,6 +71,12 @@ usebindportip = true
 #
 swarmMode = false
 
+# Define a default docker network to use for connections to all containers.
```

Comment:
Maybe we should add the fact that it's possible to override this with the label `traefik `traefik.docker.network`

Reply:
done